data/tokens.json

```diff
@@ -2541,6 +2541,56 @@
       }
     }
   },
+  "link": {
+    "gtmds_link_margin": {
+      "value": "{gtmds_size-sm}",
+      "type": "spacing"
+    },
+    "gtmds_link_padding-horizontal": {
+      "value": "{gtmds_size-xs}",
+      "type": "spacing"
+    },
+    "gtmds_link_color": {
+      "value": "{Primary.gtmds_color_primary-8}",
+      "type": "color"
+    },
+    "gtmds_link_color-hover": {
+      "value": "{Primary.gtmds_color_primary-10}",
+      "type": "color"
+    },
+    "gtmds_link_color-disabled": {
+      "value": "{Text.gtmds_color_text-color-subdued-disabled}",
+      "type": "color"
+    },
+    "gtmds_link_border-radius-focus": {
+      "value": "{gtmds_border-radius-xs}",
+      "type": "borderRadius"
+    },
+    "gtmds_link_typography-sm": {
+      "value": "{paragraph_sm-accented}",
+      "type": "typography"
+    },
+    "gtmds_link_typography-md": {
+      "value": "{paragraph_md-accented}",
+      "type": "typography"
+    },
+    "gtmds_link_typography-lg": {
+      "value": "{paragraph_lg-accented}",
+      "type": "typography"
+    },
+    "gtmds_link_border-icon-sm": {
+      "value": "{gtmds_icon_border-width-sm}",
+      "type": "borderWidth"
+    },
+    "gtmds_link_border-icon-md": {
+      "value": "{gtmds_icon_border-width-md}",
+      "type": "borderWidth"
+    },
+    "gtmds_link_border-icon-lg": {
+      "value": "{gtmds_icon_border-width-md}",
+      "type": "borderWidth"
+    }
```

Comment:
now I'm thinking do we really need these? what if we used the general tokens directly on the link? not sure if we need to define component-specific ones for a link as it is not exactly a component but more like some sort of a primitive

Reply:
@StefanIvanov regarding the border I agree with you, but I can't entirely agree with the typography. The link may be primitive, but it is a component as well since it holds some styling and states. We need an abstraction, in order to control this better.

Reply:
I agree 100% but then shouldn't we reconsider the typographies so that the underline styling is part of the value set?

Reply:
This really depends on the approach we want to adopt. Whether it is an underlined text or a bottom border. Both are true with their own pros and cons. 

I do prefer the border approach since you have more control over things like margin, size, length, etc.